lib/init.js

```diff
@@ -31,6 +31,19 @@ const findElixirProjectPath = async (editorPath) => {
   const editorDir = dirname(editorPath);
   const mixexsPath = find(editorDir, 'mix.exs');
   if (mixexsPath !== null) {
+    const pathArray = mixexsPath.split('/');
+    if (pathArray.length > 3 && pathArray[pathArray.length - 3] === 'apps') {
+      //  Treat this as an umbrella app. This may be wrong -
+      //  If you happen to keep your code in a directory called 'apps'
+      pathArray.splice((pathArray.length - 3), 3);
+      const umbrellaProjectPath = pathArray.join('/');
```

Comment:
@Arcanemagus I'm probably doing something wrong, but when I use `join(pathArray)` I get an exception thrown by the `assertPath()` function in `path.js`. Seems like the array of path components is being treated as an array containing an array.

This is what the debugger reports:

* The `pathArray` variable being passed into `join()` is: `pathArray = ["", "Users", "mkelly",...`
* The `arguments` variable inside the `join()` function is: `arguments = [Array[7]]`

Seems like it's being wrapped in another array somehow. Should I be "exploding" or "splatting" the array on the way into `join()`